src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/CommonPageElements.java

```diff
@@ -0,0 +1,195 @@
+            + "/following-sibling::ul[@class='dropdown-menu']/li/"
+            + "a[text()='Support']")
+    WebElement subMenuFromHeaderSupportItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Dates']")
+    WebElement subMenuFromHeaderDatesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Search']")
+    WebElement subMenuFromHeaderSearchItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='complex-table.html']")
+    WebElement subMenuFromHeaderComplexTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='simple-table.html']")
+    WebElement subMenuFromHeaderSimpleTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='user-table.html']")
+    WebElement subMenuFromHeaderUserTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='table-pages.html']")
+    WebElement subMenuFromHeaderTableWithPagesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']"
+            + "/li/a[@href='different-elements.html']")
+    WebElement subMenuFromHeaderDifferentElementsItem;
+
+    @FindBy(css = "ul.sidebar-menu.left >li[index='3']")
+    WebElement leftServiceMenu;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Support']")
+    WebElement subMenuFromLeftSupportItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Dates']")
+    WebElement subMenuFromLeftDatesItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Search']")
+    WebElement subMenuFromLeftSearchItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='complex-table.html']")
+    WebElement subMenuFromLeftComplexTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='simple-table.html']")
+    WebElement subMenuFromLeftSimpleTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='user-table.html']")
+    WebElement subMenuFromLeftUserTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']"
+            + "//a[@href='table-pages.html']")
+    WebElement subMenuFromLeftTableWithPagesItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']"
+            + "//a[@href='different-elements.html']")
+    WebElement subMenuFromLeftDifferentElementsItem;
```

Comment:
Looks like that most of these elements extra here and could be deleted

Reply:
removed